clusters/app.ci/supplemental-ci-images/validation-images/yaml/yamllint.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: yamllint
+  namespace: ci
+---
+kind: BuildConfig
```

Comment:
Do we also need a periodic trigger for this build, like https://github.com/openshift/release/blob/a2f65285a6502d9b3994c3dada40422edf2cad5e/ci-operator/jobs/infra-periodics.yaml#L2-L23 ?

Reply:
I think we are not that eager for the latest version of the tool.
Saving the BC here is to update easily if we need.
We have other BCs in app.ci. We only do manual trigger by `oc start-build`.

Reply:
'kay